### ISSUE TITLE
Mobile notification icon is a circle instead of the Immich logo

### ISSUE THREAD

**Issue by JordyEGNL** [immich-app/immich#11745]
### The bug

The Immich logo is expected (in monochrome form, white or gray) instead of a circle.
| ![signal-2024-08-13-102152_002](https://github.com/user-attachments/assets/ab902ecd-50e2-4889-8afc-2e0cdfaf9749) | ![signal-2024-08-13-102152_003](https://github.com/user-attachments/assets/ab00310d-ca7f-4f97-b91a-6f8d6777ad8c) |
| -- | -- |

### The OS that Immich Server is running on

-

### Version of Immich Server

v1.111.0

### Version of Immich Mobile App

v1.111.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

```bash
In order to get a notification from Immich I turned off my Immich server and waited for the background backup to begin.
```

### Additional information

Running Android 14

**Comment by karthikraja001** [comment]
I'll work on this issue

**Comment by JordyEGNL** [comment]
Fixed with #11863
